devOpcuaSup/open62541/install_open62541.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+#
+# This script installs the open62541 library for OPC UA support in EPICS.
+# It downloads the specified version of open62541, builds it, and configures the EPICS environment.
+# It requires curl, cmake and openssl-dev to be installed on the system.
+#
+# Usage: ./install_open62541.sh [version]
+# If no version is specified, a default version is defined in the script.
+
+# Default version of open62541 to use
+DEFAULT_VERSION="1.3.15"
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+OPEN62541_VERSION=${1:-$DEFAULT_VERSION}
+
+# Check if cmake and curl are available
+if ! command -v cmake &> /dev/null || ! command -v curl &> /dev/null; then
+    echo "Error: cmake and/or curl is not installed. Please install them and try again."
+    exit 1
+fi
+
+# Download and extract open62541 if not already present
+echo "Downloading open62541 version $OPEN62541_VERSION..."
+curl -L -o "$SCRIPT_DIR/open62541-$OPEN62541_VERSION.tar.gz" "https://github.com/open62541/open62541/archive/refs/tags/v$OPEN62541_VERSION.tar.gz"
+if [ $? -ne 0 ]; then
+    echo "Error: Failed to download open62541 version $OPEN62541_VERSION."
+    exit 1
+fi
+temp_dir="`mktemp -d -p "$SCRIPT_DIR"`"
```

Comment:
Need to remove `$temp_dir` and the install dir to be sure to start from a clean situation.

Reply:
Ah, I didn't see `mktemp`.

I would use a fix name and remove the dir before building.
The `mktemp` is only important for parallel make etc. where two builds might be executing at the same time in the same location.

Reply:
See above. I didn't realize the `mktemp`.
I would do it in a fixed location, clean before build and leave the dir in case of error.